README.md

```diff
@@ -292,6 +293,48 @@ namespace LoggerEventFlow
 }
 ```
 
+#### ETW (Event Tracing for Windows)
+
+*Nuget package:* [**Microsoft.Diagnostics.EventFlow.Inputs.Etw**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Inputs.Etw/)
+
+This input captures data from Microsoft Event Tracing for Windows (ETW) providers. Both manifest-based providers as well as providers based on managed `EventSource` infrastructure are supported. The data is captured machine-wide and requires that the identity the process uses belongs to Performance Log Users built-in administrative group.
+
+*Note* 
+
+To capture data from EventSources running in the same process as EventFlow, the [EventSource input](#eventsource) is a better choice, with better performance and no additional security requirements.
+
+*Configuration example*
+```json
+{
+    "type": "ETW",
+    "providers": [
+        {
+            "providerName": "Microsoft-ServiceFabric",
+            "level": "Warning",
+            "keywords": "0x7F"
+        }
+    ]
+}
+```
+
+*Top Object*
+
+| Field | Values/Types | Required | Description |
+| :---- | :-------------- | :------: | :---------- |
+| `type` | "ETW" | Yes | Specifies the input type. For this input, it must be "ETW". |
+| `providers` | JSON array | Yes | Specifies ETW providers to collect data from. |
+
+*Providers object*
+
+| Field | Values/Types | Required | Description |
+| :---- | :-------------- | :------: | :---------- |
+| `providerName` | provider name | Yes(*) | Specifies the name of the ETW provider to track. |
+| `providerGuid` | provider GUID | Yes(*) | Specifies the GUID of the ETW provider to track. |
+| `level` | Critial, Error, Warning, Informational, Verbose, LogAlways | No | Specifies the collection trace level. Traces with equal or higher severity than specified are collected. For example, if Warning is specified, then Critial, Error, and Warning traces are collected. Default is All. |
```

Comment:
Well, yes an no. Yes, LogAlways is 0 and Critical is 1. But LogAlways has a special meaning, which is "provider decides which events are raised" and by default it results in raising (and capturing) all events. 

I have updated the docs.